insights_client/constants.py

```diff
@@ -12,23 +12,25 @@ class InsightsConstants(object):
     package_path = os.path.dirname(
         os.path.dirname(os.path.abspath(__file__)))
     sleep_time = 300
-    user_agent = app_name + '/' + version
-    default_conf_dir = '/etc/' + app_name + '/'
-    log_dir = '/var/log/' + app_name
-    default_log_file = log_dir + '/' + app_name + '.log'
-    default_conf_file = default_conf_dir + app_name + '.conf'
-    default_sed_file = default_conf_dir + '.exp.sed'
-    default_ca_file = default_conf_dir + 'cert-api.access.redhat.com.pem'
+    user_agent = os.path.join(app_name, 'version')
+    default_conf_dir = os.path.join(os.sep, 'etc', app_name)
+    log_dir = os.path.join(os.sep, 'var', 'log', app_name)
+    default_log_file = os.path.join(log_dir, app_name) + '.log'
```

Comment:
Why do we use "+" concatenation here and os.path.join() in other places? Maybe just a styling preference and not really much functionality difference.